selfdrive/car/tests/test_lateral_limits.py

```diff
@@ -32,7 +32,7 @@
 car_model_jerks: DefaultDict[str, Dict[str, float]] = defaultdict(dict)
 
 
-@parameterized_class('car_model', [(c,) for c in CAR_MODELS])
+@parameterized_class('car_model', [(c,) for c in sorted(CAR_MODELS)])
```

Comment:
for xdist, the test ordering has to be deterministic

Reply:
good catch!